editable-grid-parent/editable-grid-examples/pom.xml

```diff
@@ -20,6 +20,35 @@
 			<groupId>${project.groupId}</groupId>
 			<artifactId>wicketstuff-editable-grid</artifactId>
 		</dependency>
+
+		<!-- LOGGING DEPENDENCIES - SLF4J-SIMPLE -->
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-simple</artifactId>
+			<version>${slf4j.version}</version>
+		</dependency>
+
+		<!--  JUNIT DEPENDENCY FOR TESTING -->
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<version>5.9.3</version>
```

Comment:
please remove versions from here (they are defined in `dependencyManagement` at parent pom